test/testcafe/spec/EnrollAuthenticatorEmailFirst_spec.js

```diff
@@ -7,18 +7,21 @@ import { checkConsoleMessages } from '../framework/shared';
 import emailAuthenticatorPreEnrollData from '../../../playground/mocks/data/idp/idx/authenticator-enroll-email-first';
 import emailVerification from '../../../playground/mocks/data/idp/idx/authenticator-enroll-email-first-emailmagiclink-true';
 
-fixture('Pre-Enroll Email Authenticator Form');
+fixture('Pre-Enroll Email Authenticator Form').meta('v3', true);
 
 async function setup(t) {
   const challengeEmailPageObject = new ChallengeEmailPageObject(t);
   await challengeEmailPageObject.navigateToPage();
+  await challengeEmailPageObject.formExists();
   return challengeEmailPageObject;
 }
 
 const sendEmailMock = RequestMock()
   .onRequestTo('http://localhost:3000/idp/idx/introspect')
   .respond(emailAuthenticatorPreEnrollData)
   .onRequestTo('http://localhost:3000/idp/idx/credential/enroll')
+  .respond(emailVerification)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/poll')
```

Comment:
I added this because the original remediation has a poll apart of it, if the test takes too long, that poll could occur and fail the test. That could result in flaky tests, adding this here to return the same response resolves this issue.

Reply:
Nice! Thanks for making this test less flaky <3